brianlong/www.validators.app
# 179433806 fix cluster stats

#### What's this PR do?
all stats should be visible and should be calculated correctly

#### How should this be manually tested?
- check all the tabs in cluster-stats view, all tables should be populated and data should match the index page

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/179433806

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
